src/boot/ksdk1.1.0/devBMX055.h

```diff
@@ -1,64 +1,69 @@
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
 */
-void		initBMX055accel(const uint8_t i2cAddress, uint16_t operatingVoltageMillivolts);
-void		initBMX055gyro(const uint8_t i2cAddress, uint16_t operatingVoltageMillivolts);
-void		initBMX055mag(const uint8_t i2cAddress, uint16_t operatingVoltageMillivolts);
+void initBMX055accel(const uint8_t i2cAddress, uint16_t operatingVoltageMillivolts);
+void initBMX055gyro(const uint8_t i2cAddress, uint16_t operatingVoltageMillivolts);
+void initBMX055mag(const uint8_t i2cAddress, uint16_t operatingVoltageMillivolts);
 
-WarpStatus	writeSensorRegisterBMX055accel(uint8_t deviceRegister, uint8_t payload, uint16_t menuI2cPullupValue);
-WarpStatus	writeSensorRegisterBMX055gyro(uint8_t deviceRegister, uint8_t payload, uint16_t menuI2cPullupValue);
-WarpStatus	writeSensorRegisterBMX055mag(uint8_t deviceRegister, uint8_t payload, uint16_t menuI2cPullupValue);
+WarpStatus writeSensorRegisterBMX055accel(uint8_t deviceRegister, uint8_t payload);
+WarpStatus writeSensorRegisterBMX055gyro(uint8_t deviceRegister, uint8_t payload);
+WarpStatus writeSensorRegisterBMX055mag(uint8_t deviceRegister, uint8_t payload);
 
-WarpStatus	configureSensorBMX055accel(uint8_t payloadPMU_RANGE,
-					uint8_t payloadACCD_HBW,
-					uint16_t menuI2cPullupValue);
-WarpStatus	configureSensorBMX055gyro(uint8_t payloadRANGE,
-					uint8_t payloadBW,
-					uint8_t payloadLPM1,
-					uint8_t payloadRATE_HBW,
-					uint16_t menuI2cPullupValue);
-WarpStatus	configureSensorBMX055mag(uint8_t payloadPowerCtrl,
-					uint8_t payloadOpMode,
-					uint16_t menuI2cPullupValue);
+WarpStatus configureSensorBMX055accel(uint8_t payloadPMU_RANGE,
+                                      uint8_t payloadACCD_HBW);
+WarpStatus configureSensorBMX055gyro(uint8_t payloadRANGE,
+                                     uint8_t payloadBW,
+                                     uint8_t payloadLPM1,
+                                     uint8_t payloadRATE_HBW);
+WarpStatus configureSensorBMX055mag(uint8_t payloadPowerCtrl,
+                                    uint8_t payloadOpMode);
 
-WarpStatus	readSensorRegisterBMX055accel(uint8_t deviceRegister, int numberOfBytes);
-WarpStatus	readSensorRegisterBMX055gyro(uint8_t deviceRegister, int numberOfBytes);
-WarpStatus	readSensorRegisterBMX055mag(uint8_t deviceRegister, int numberOfBytes);
+WarpStatus readSensorRegisterBMX055accel(uint8_t deviceRegister, int numberOfBytes);
```

Comment:
This doesn't match the alignment convention (whether we agree on tabs versus spaces or not).